File: CotuTalk_Program/app/src/main/java/com/example/cotutalk_program/AcessoAPI/data/Postagem.kt
package com.example.cotutalk_program.AcessoAPI.data

import com.google.gson.annotations.SerializedName

data class Postagem (
    @SerializedName("idPostagem")
    val IdPostagem : Int,
    @SerializedName("idUsuario")
    val IdUsuario : Int,
    @SerializedName("idGrupo")
    val IdGrupo : Int,
    @SerializedName("conteudo")
    val Conteudo: String
//    val Grupo: Grupo,
//    val Usuario: Usuario
)

File: api/TWTodos/Models/Postagem.cs
using System.Text.Json.Serialization;
public class Postagem
{
    public int IdPostagem { get; set; }
    public int IdUsuario { get; set; }
    public int IdGrupo { get; set; }
    public required Usuario Usuario { get; set; }
    public required Grupo Grupo { get; set; }
    public required string Conteudo { get; set; }
    [JsonIgnore]
    public ICollection<Curtida> Curtidas { get; set; } = new List<Curtida>();
    [JsonIgnore]
    public ICollection<Resposta> Respostas { get; set; } = new List<Resposta>();
}